### PR TITLE
log out url for debugging and return model from init

### PR DESCRIPTION
**Pull request by aintgoin2goa**

### CODE REVIEW

File: server/fetchFlagJson.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ function initialFetch(opts){
 				throw new Error('Bad Response from Flags status=' + result.response.status);
 			}
 			options.url = result.request.url;
+			console.log('Flags loaded from %s', result.request.url);
```
Comment (matthew-andrews) [inline server/fetchFlagJson.js]: worht putting a `blah=` before the `%s` …… maybe `source=`? to make splunk search easier?

Comment (aintgoin2goa) [inline server/fetchFlagJson.js]: good idea - will add a bunch of splunk-friendly stuff

**Comment by matthew-andrews** [comment]
:+1: :ship:

File: server/fetchFlagJson.js [hunk 2]
```diff
@@ -23,6 +23,7 @@ function initialFetch(opts){
 				throw new Error('Bad Response from Flags status=' + result.response.status);
 			}
 			options.url = result.request.url;
+			console.log('event=FLAGS_LOADED source=%s', result.request.url);
```
Comment (ironsidevsquincy) [inline server/fetchFlagJson.js]: https://github.com/Financial-Times/next-logger ?

**Pull request merged**